CHANGELOG/v1.7.0-beta.1.md

```diff
@@ -0,0 +1,411 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
+
+* Enable kubeadm feature gates mutation
```

Comment:
We probably want to write some more here when we get closer to the release (I currently just don't have the time to go through all the changes below to write something up)

Likely candidates are the changes below New Features